media/css/mozorg/home/brand-takeover.scss

```diff
@@ -0,0 +1,181 @@
+    background: #fff;
+    box-shadow: 0 3px 3px rgba(0, 0, 0, .5);
+    color: #000;
+    left: 0;
+    min-width: $width-phone;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    z-index: 999;
+
+    @include transition(background-color 450ms ease, color 450ms ease);
+
+    &.dark {
+        background: #000;
+        color: #fff;
+    }
+
+    .takeover-content {
+        text-align: center;
+        padding: 40px 0;
+    }
+
+    .takeover-cta {
+        @include font-size(20px);
+        margin-top: 1em;
+
+        a {
+            @include transition(background-color 150ms ease-out);
+            background-color: #ff8c8e;
+            color: #000;
+            padding: 0 2px;
+            text-decoration: none;
+
+            &:hover,
+            &:focus,
+            &:active {
+                background-color: darken(#ff8c8e, 10%);
+                text-decoration: underline;
+            }
+        }
+    }
+
+    .video-frame {
+        height: 169px;
+        margin: 40px auto;
+        max-width: 1280px;
+        position: relative;
+        text-align: center;
+        width: 300px;
+
+        iframe {
+            width: 100%;
+            height: 100%;
+        }
+
+        @media #{$mq-tablet} {
+            width: 700px;
+            height: 394px;
+        }
+
+        @media #{$mq-desktop} {
+            width: 960px;
+            height: 540px;
+        }
+
+        @media #{$mq-desktop-wide} and (min-height: 800px) {
+            width: 1280px;
+            height: 720px;
+        }
+    }
+
+    .header-strip {
+        @include clearfix;
+        background-color: #ff8c8e;
+        padding: 10px;
+        position: relative;
+
+        .content {
+            padding: 0;
+            margin: 0 auto;
+
+            @media #{$mq-tablet} {
+                padding: 0 20px;
+            }
+        }
+
+        img {
+            float: left;
+        }
+
+        button {
+            @include font-size(14px);
+            @include trailing-arrow;
+            background: transparent none;
+            border: 0;
+            color: #000;
+            cursor: pointer;
+            float: right;
+            height: 36px;
+            padding-right: 10px;
+            position: relative;
+            text-align: right;
+            text-decoration: none;
+            text-transform: uppercase;
+
+
+            &:after {
+                @include transition(right 150ms ease-in);
+                position: absolute;
+                right: 0;
+            }
+
+            &:hover,
+            &:focus,
+            &:active {
+                text-decoration: underline;
+
+                &:after {
+                    right: -4px;
+                }
+            }
+        }
+    }
+}
+
+.page-overlay {
+    background-color: #666;
+    background-color: rgba(0, 0, 0, .7);
+    bottom: 0;
+    height: 100%;
+    left: 0;
+    position: fixed;
+    right: 0;
+    top: 0;
+    width: 100%;
+    z-index: 998;
```

Comment:
I think the takeover itself is 999, so the page overlay behind is it 998. So why 999 instead of 1000? Because every byte counts!